cni/network/network.go

```diff
@@ -742,6 +771,13 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	if opt.resultV6 != nil {
+		// inject ipv6 routes to Linux pod if ipamMode is dual stack overlay
+		ipamMode := string(util.IpamMode(opt.nwCfg.IPAM.Mode))
+		if ipamMode == string(util.DualStackOverlay) {
```

Comment:
because only if ipamMode is dualStackOverlay mode, then we set epInfo.IPV6Mode to ipamMode as "dualStackOverlay", otherwise if it's v4Overlay, we will still set epInfo.IPV6Mode to v4Overlay which is wrong

Reply:
how come ipammode will be v4Overlay if resultv6 is not nil?